packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -40,6 +40,11 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
       enabled: !!handle,
     },
   );
+
+  if (!squad && isLoading) return <SquadLoading />;
+
+  if (isFallback || !squad.active) return <Unauthorized />;
+
```

Comment:
I think the correct way to go is to set the `enabled` property to `true` until the conditions above are reversely met. From the docs it is ideal to not return early when there are hooks declared under the condition.
> Instead, always use Hooks at the top level of your React function, before any early returns. 

Reference: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Ah I see, well we can modify the order.
I assumed we wouldn't need it, but yes good point.